### PR TITLE
fix Issue 17275 - [REG 2.072.0] AssertError@declaration.d(2121): Asse…

### PR DESCRIPTION
**Pull request by WalterBright**
…rtion failure

### CODE REVIEW

File: test/fail_compilation/test17275.d [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* TEST_OUTPUT:
+---
+fail_compilation/test17275.d(12): Error: undefined identifier 'ModuleGroup', did you mean function 'moduleGroup'?
+fail_compilation/test17275.d(12): Error: inout on return means inout must be on a parameter as well for inout(ModuleGroup)()
```
Comment (ZombineDev) [inline test/fail_compilation/test17275.d]: There's already test/fail_compilation/fail17275.d (https://github.com/dlang/dmd/pull/6690/files#diff-278b7365e40009be458ef32b7f8d85a3). Why don't you put the test output in there?

**Review by ibuclaw — CHANGES_REQUESTED** [review]
As per comment (I actually made such a commit in #6746 here https://github.com/ibuclaw/dmd/commit/407d2a5c3d068243c2899ef95f079170baa91178)

**Comment by WalterBright** [comment]
@ibuclaw I don't understand what you're asking for.

**Comment by ibuclaw** [comment]
@WalterBright - you don't need a new test case, you just need to copy the `TEST_OUTPUT` to  test/fail_compilation/fail17275.d

**Comment by WalterBright** [comment]
> @WalterBright - you don't need a new test case, you just need to copy the TEST_OUTPUT to test/fail_compilation/fail17275.d

Well, fail17275 was in another branch, and now I've fouled it up completely trying to merge the two.

**Comment by ibuclaw** [comment]
How did it end up in another branch?

**Pull request closed without merging**